include/xlnt/utils/exceptions.hpp

```diff
@@ -240,9 +240,9 @@ class XLNT_API invalid_attribute : public exception
 {
 public:
     /// <summary>
-    /// Default constructor.
+    /// Constructor.
     /// </summary>
-    invalid_attribute();
+    explicit invalid_attribute(const char *optional_message = nullptr);
```

Comment:
just wondering, is there any reason why you switched to an explicit constructor?

Reply:
Since this is now a constructor with one parameter, I would like to avoid the compiler from allowing implicit conversions. To be precise, I would like to avoid [such issues](https://stackoverflow.com/a/121216).